fk/teamcreate.go

```diff
@@ -139,8 +154,25 @@ func teamCreate() exitCode {
 
 	printHeader("Finishing setup")
 
-	out.Print("You've indicated you want setup " + teamName + " using your key\n")
-	out.Print(fmt.Sprintf("%s\n", key.Fingerprint()))
+	uuid, err := uuid.NewV4()
+	if err != nil {
+		out.Print(ui.FormatFailure("Error creating UUID for team", nil, err))
+		return 1
+	}
+
+	t := team.Team{
+		UUID:   uuid,
+		Name:   teamName,
+		People: teamMembers,
+	}
+
+	printCheckboxPending("Create team roster")
+	_, err = t.Save(fluidkeysDirectory)
+	if err != nil {
+		printCheckboxFailure("Create team roster", err)
+	}
+	printCheckboxSuccess("Create team roster at " +
```

Comment:
This was "Create team roster *in*" in the [design doc](https://docs.google.com/document/d/1FHQXHP7X8Uvtvn72OqZNVYyFMocrJ2MM1zWtYTM4d0c/edit#heading=h.n4nbzbgpaffm)

Reply:
Hawk eyes! 👁 💪